modules/moa-installing-prerequisites.adoc

```diff
@@ -0,0 +1,216 @@
+** This user will leverage the IAM credentials you provide when configuring the AWS cli in the next section.
+. Install and configure the AWS cli
+.. Install the link:https://aws.amazon.com/cli/[aws-cli].
+.. Configure the `aws-cli` to use the AWS account that you would like to deploy your cluster into. .... Modify your `~/.aws/credentials` file to specify the correct `aws_access_key_id` and `aws_secret_access_key`.
++
+[source, bash]
+----
+$ cat ~/.aws/credentials
+[default]
+aws_access_key_id = <my-aws-access-key-id>
+aws_secret_access_key = <my-aws-secret-access-key>
+----
++
+.. Modify your `~/.aws/config` file to specify the AWS region you want to use.
++
+[source, bash]
+----
+$ cat ~/.aws/config
+[default]
+output = table
+region = us-east-2
+----
++
+.. To verify your configuration, run the following command to query the AWS api:
++
+[source, bash]
+----
+$ aws ec2 describe-regions
+---------------------------------------------------------------------------------
+|                                DescribeRegions                                |
++-------------------------------------------------------------------------------+
+||                                   Regions                                   ||
+|+-----------------------------------+-----------------------+-----------------+|
+||             Endpoint              |      OptInStatus      |   RegionName    ||
+|+-----------------------------------+-----------------------+-----------------+|
+||  ec2.eu-north-1.amazonaws.com     |  opt-in-not-required  |  eu-north-1     ||
+||  ec2.ap-south-1.amazonaws.com     |  opt-in-not-required  |  ap-south-1     ||
+||  ec2.eu-west-3.amazonaws.com      |  opt-in-not-required  |  eu-west-3      ||
+||  ec2.eu-west-2.amazonaws.com      |  opt-in-not-required  |  eu-west-2      ||
+||  ec2.eu-west-1.amazonaws.com      |  opt-in-not-required  |  eu-west-1      ||
+||  ec2.ap-northeast-2.amazonaws.com |  opt-in-not-required  |  ap-northeast-2 ||
+||  ec2.ap-northeast-1.amazonaws.com |  opt-in-not-required  |  ap-northeast-1 ||
+||  ec2.sa-east-1.amazonaws.com      |  opt-in-not-required  |  sa-east-1      ||
+||  ec2.ca-central-1.amazonaws.com   |  opt-in-not-required  |  ca-central-1   ||
+||  ec2.ap-southeast-1.amazonaws.com |  opt-in-not-required  |  ap-southeast-1 ||
+||  ec2.ap-southeast-2.amazonaws.com |  opt-in-not-required  |  ap-southeast-2 ||
+||  ec2.eu-central-1.amazonaws.com   |  opt-in-not-required  |  eu-central-1   ||
+||  ec2.us-east-1.amazonaws.com      |  opt-in-not-required  |  us-east-1      ||
+||  ec2.us-east-2.amazonaws.com      |  opt-in-not-required  |  us-east-2      ||
+||  ec2.us-west-1.amazonaws.com      |  opt-in-not-required  |  us-west-1      ||
+||  ec2.us-west-2.amazonaws.com      |  opt-in-not-required  |  us-west-2      ||
+|+-----------------------------------+-----------------------+-----------------+|
+----
++
+. Install `moactl`, the {product-title} command line utility.
```

Comment:
Should we explain what moactl is at the beginning of this getting started guide? I got to this step and was like "Oh, that's what maoctl is!", and then I was like wait did we talk about it already. It's mentioned twice above, but never outlined until this step. So the user doesn't know what moactl is until step 3 despite the previous mentions.

Reply:
Good call. I added this line to the end of the very first sentence in the module: "...the {product-title} command line utility."